src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+  application:
+    name: case-validation-server
+
+server:
+  port: 5010
```

Comment:
port 5010 will already be used by balances-adjustment-server, so you have to choose another port number